### PR TITLE
feat: Added support for choices operator

### PR DESCRIPTION
**Pull request by abhilash-kumar-nair**

### CODE REVIEW

File: modelon/impact/client/experiment_definition.py [hunk 1]
```diff
@@ -101,6 +101,32 @@ def __str__(self):
         return f"range({self.start_value},{self.end_value},{self.no_of_steps})"
 
 
+class Choices(Operator):
+    """
+    Choices operator class for parameterizing custom batch runs.
```
Comment (efredriksson-modelon) [inline modelon/impact/client/experiment_definition.py:106]: remove 'custom'

File: modelon/impact/client/experiment_definition.py [hunk 2]
```diff
@@ -101,6 +101,32 @@ def __str__(self):
...
+            Variable number of numerical arguments to sweep.
+
+    Examples::
+
+        from modelon.impact.client import Choices
+
+        fmu = model.compile().wait()
+        experiment_definition = fmu.new_experiment_definition(
+            custom_function).with_modifiers({'inertia1.J': 2,
+            'inertia2.J': Choices(0.1, 0.5)}, k=2, w=7)
```
Comment (efredriksson-modelon) [inline modelon/impact/client/experiment_definition.py:120]: maybe just `({'inertia2.J': Choices(0.1, 0.5)})` to make the exmple easier to grasp?

File: docs/source/tutorial.rst [hunk 3]
```diff
@@ -162,6 +162,21 @@ Here the ``Range()`` operator class is used to specify the ``start_value``, ``en
 The parametrization in the examples above would configure a set of three simulation cases for the fmu with ``3`` equidistant
 values set for the ``PI.k`` parameter, with a start value of ``10`` and an end value of ``100``.
 
+It is also possible to create multi-execution experiments with custom parameter values to sweep::
```
Comment (efredriksson-modelon) [inline docs/source/tutorial.rst:165]: `custom` -> `an explicit list (choices) of`

File: docs/source/tutorial.rst [hunk 4]
```diff
@@ -162,6 +162,21 @@ Here the ``Range()`` operator class is used to specify the ``start_value``, ``en
...
+
+   from modelon.impact.client import Choices
+
+   experiment_definition = experiment_definition.with_modifiers({'PI.k': Choices(10, 20, 30, 40)})
+
+Here the ``Choices()`` operator class is used to specify custom values for the parameter to sweep. The parametrization in the
```
Comment (efredriksson-modelon) [inline docs/source/tutorial.rst:171]: `custom` -> `an explicit list of`

File: docs/source/tutorial.rst [hunk 5]
```diff
@@ -162,6 +162,21 @@ Here the ``Range()`` operator class is used to specify the ``start_value``, ``en
...
 values set for the ``PI.k`` parameter, with a start value of ``10`` and an end value of ``100``.
 
+It is also possible to create multi-execution experiments with custom parameter values to sweep::
+
+   from modelon.impact.client import Choices
+
+   experiment_definition = experiment_definition.with_modifiers({'PI.k': Choices(10, 20, 30, 40)})
+
+Here the ``Choices()`` operator class is used to specify custom values for the parameter to sweep. The parametrization in the 
+examples above would configure a set of four simulation cases for the fmu with ``4`` custom values for the ``PI.k`` parameter.
```
Comment (efredriksson-modelon) [inline docs/source/tutorial.rst:172]: custom -> choosen

File: docs/source/tutorial.rst [hunk 6]
```diff
@@ -162,6 +162,21 @@ Here the ``Range()`` operator class is used to specify the ``start_value``, ``en
...
+
+It also possible to have use a combination of range and choices operator to setup a batch run::
```
Comment (efredriksson-modelon) [inline docs/source/tutorial.rst:174]: remove 'have'

**Review by efredriksson-modelon — CHANGES_REQUESTED** [review]
Nice addition 👍

Minor, but I think the documentation can be clarified by being more precise on what 'custom' here means.

**Comment by abhilash-kumar-nair** [comment]
> Nice addition 👍
> 
> Minor, but I think the documentation can be clarified by being more precise on what 'custom' here means.

@efredriksson-modelon , Thanks for the review! :) I have made the suggested the changes. Could you please take another look?

**Review by efredriksson-modelon — APPROVED** [review]
👍

**Pull request merged**